template.json

```diff
@@ -3,10 +3,13 @@
     "contentVersion": "1.0.0.0",
     "parameters": {
         "Name": {
-            "type": "String"
+            "type": "String",
+            "defaultValue": "AlertlogicCollector"
         },
         "Storage Name": {
-            "type": "String"
+            "type": "String",
+            "defaultValue": "alcollectorstorage"
```

Comment:
I'm not sure it's a good idea to set a default value for this as SA needs to be globally unique in all of Azure afaik.  https://docs.microsoft.com/en-us/azure/architecture/best-practices/naming-conventions  
After the first one is created the default value will just be a pain?